packages/ag-charts-community/src/chart/chartHeapMemory.test.ts

```diff
@@ -122,7 +122,7 @@ describe.skip('Chart Heap Memory', () => {
             const heapProportionChange = Math.abs(endingHeap - startingHeap) / startingHeap;
 
             // console.log({ startingHeap, endingHeap, heapProportionChange });
-            expect(heapProportionChange).toBeLessThan(0.2);
+            expect(heapProportionChange).toBeLessThan(0.3);
```

Comment:
There is an amount of extra memory used, typically up to about `0.25` on my machine. The graph vertices and edges are all cleared after it is resolved. This is not unexpected.

Reply:
I also have a change that should improve stability of this test (it will get run in series rather than parallel with other tests): https://github.com/ag-grid/ag-charts/pull/4339

Reply:
Yep, that fixed it, thanks.